src/Domain/Creators/AlignedLattice.hpp

```diff
@@ -18,7 +18,7 @@
 namespace domain {
 namespace creators {
 
-/// \ingroup DomainCreatorsGroup
+/// \ingroup ComputationalDomainGroup
```

Comment:
I personally would prefer the domain creators just be in their namespace and not also in the `ComputationalDomainGroup`. The reason is that since they're in their own namespace and a fairly separate part of the domain we can keep the `ComputationalDomainGroup` section easier to browse by having the creators be separate. What do you think?

Reply:
Actually I almost did it as you are suggesting.. I'll go ahead and make the change. 🙂

Reply:
I think only the namespace `domain::creators` and the base class `DomainCreator` should be in the `ComputationalDomainGroup` and not the derived classes

Reply:
I don't have an opinion on the base class. What's your reason for it being in the group? :)

Reply:
it isn't in the namespace

Reply:
welp... Yep... 

A "not for this PR" question: should we maybe just have the base classes in these cases in the namespace too?